agents/labjackT7

```diff
@@ -0,0 +1,215 @@
+from ocs import ocs_agent, site_config, client_t
+import random
+import time
+import threading
+import os
+import struct
+from pymodbus.client.sync import ModbusTcpClient
+
+from ocs.ocs_twisted import TimeoutLock
+
+from autobahn.wamp.exception import ApplicationError
```

Comment:
You can clean up the imports a bit, a bunch of these aren't used:
* random
* threading
* os
* ApplicationError
* client_t